From de002f503834d2e25b9428f89054dcf61c1ba549 Mon Sep 17 00:00:00 2001
From: Morgane BUTSCHER <morganex.butscher@intel.com>
Date: Fri, 16 Nov 2012 16:13:48 +0100
Subject: [PORT FROM MAIN] UEVENTD: /dev/mdmTRace rights should be changed

BZ: 63993

/dev/mdmTrace is used for the logging over HSI.

/dev/mdmTrace should belong to the radio group.

Change-Id: I006d7c7b20532ec1f7ed5496b675e82953b9bb01
Orig-Change-Id: I1e6df8b834d84c5e0240e902d7ec27779dbf905c
Signed-off-by: Morgane BUTSCHER <morganex.butscher@intel.com>
Reviewed-on: http://android.intel.com:8080/80207
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Ravindran, Arun <arun.ravindran@intel.com>
Reviewed-by: Lebsir, SamiX <samix.lebsir@intel.com>
Tested-by: Lebsir, SamiX <samix.lebsir@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 rootdir/ueventd.rc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index 2cf0265..686bb33 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -76,6 +76,7 @@
 /dev/mtp_usb              0660   root       mtp
 /dev/usb_accessory        0660   root       usb
 /dev/tun                  0660   system     vpn
+/dev/mdmTrace             0660   root       radio
 
 # CDMA radio interface MUX
 /dev/ts0710mux*           0640   radio      radio
-- 
1.7.4.1

